src/Window/Window.jsx

```diff
@@ -4,6 +4,8 @@ import PropTypes from 'prop-types';
 import { uniqueId } from 'lodash';
 import { Panel } from  '../index.js';
 
+import { Logger } from '../index';
```

Comment:
can be put into the above statement... but I always thought we would import from where they are actually first exported. unsure her but also very minor.

Reply:
You're so right! I've fixed imports here and in some other classes in 5ed0cc1